File: ProjecTree-was/src/main/java/com/ssafy/projectree/domain/tech/model/repository/TechStackInfoRepository.java
package com.ssafy.projectree.domain.tech.model.repository;

import com.ssafy.projectree.domain.tech.model.entity.TechStackInfo;
import org.springframework.data.jpa.repository.JpaRepository;

public interface TechStackInfoRepository extends JpaRepository<TechStackInfo, Long> {
}

File: INFERENCE/app/db/repository/tech_stack_info_repository.py
from app.db.repository.base_repository import BaseRepository
from app.db.models import TechStackInfo
from app.db.schemas.tech import TechStackInfoCreate, TechStackInfoUpdate


class TechStackInfoRepository(
    BaseRepository[TechStackInfo, TechStackInfoCreate, TechStackInfoUpdate]
):
    pass


tech_stack_info_repository = TechStackInfoRepository(TechStackInfo)
